x-pack/plugins/maps/public/embeddable/map_embeddable.tsx

```diff
@@ -221,10 +232,22 @@ export class MapEmbeddable
   }
 
   onUpdate() {
+    if (
+      this.input.filterByMapExtent !== undefined &&
+      this._prevFilterByMapExtent !== this.input.filterByMapExtent
+    ) {
+      this._prevFilterByMapExtent = this.input.filterByMapExtent;
+      if (this.input.filterByMapExtent) {
+        this.setMapExtentFilter();
```

Comment:
how do we handle updates to the extent filter ? seems we only call this when we set/unset filterByMapExtend from here and not when center point changed ?

Reply:
found it below